### DISCUSSION TITLE
Question Mania

### DISCUSSION THREAD

**Discussion by Devansh-Bots** [Devansh-Bots/Mdisk_downloader-main#4] [Q&A] [answered]
electronic gain enthalpy increases when?

**Comment by noob-mukesh** [comment, marked as answer]
Electronic gain enthalpy refers to the enthalpy change that occurs when an electron is gained by an atom or ion in the gaseous state. The electronic gain enthalpy can increase when the atom or ion has a higher affinity for gaining an electron.

**Reply by Devansh-Bots** [reply]
Good work qt
